packager/hls/base/simple_hls_notifier.cc

```diff
@@ -288,9 +288,11 @@ SimpleHlsNotifier::SimpleHlsNotifier(const HlsParams& hls_params)
       hls_params.default_text_language.empty()
           ? hls_params.default_language
           : hls_params.default_text_language;
+  bool is_independent_segments = hls_params.is_independent_segments;
```

Comment:
This intermediate variable is unnecessary.

Reply:
Removed this line